mvpoly/src/prime.rs

```diff
@@ -346,6 +346,55 @@ impl<F: PrimeField, const N: usize, const D: usize> Add<&Dense<F, N, D>> for &De
     }
 }
 
+// Subtraction
+impl<F: PrimeField, const N: usize, const D: usize> Sub for Dense<F, N, D> {
+    type Output = Self;
+
+    fn sub(self, other: Self) -> Self {
+        let mut result = Dense::new();
```

Comment:
Should this be called `zero` to make it more clear?

Reply:
Changed into a functional trait in https://github.com/o1-labs/proof-systems/pull/2496. I suppose it will close this thread.